tests/python/contrib/test_hexagon/metaschedule_e2e/export_models.py

```diff
@@ -34,14 +37,16 @@ def export_resnet50_fp16():
     mod, params = relay.frontend.from_pytorch(script_module, input_shapes)
     mod = relay.transform.ToMixedPrecision("float16")(mod)
 
-    with open("resnet50_fp16.json", "w") as fo:
-        fo.write(tvm.ir.save_json(mod))
+    with open("resnet50_fp16.json", "w") as file:
+        file.write(tvm.ir.save_json(mod))
 
-    with open("resnet50_fp16.params", "wb") as fo:
-        fo.write(relay.save_param_dict(params))
+    with open("resnet50_fp16.params", "wb") as file:
+        file.write(relay.save_param_dict(params))
```

Comment:
I think it is ok to add `# pylint: disable=invalid-name` to avoid these changes. We use that everywhere.

Reply:
I reversed them back to global variables, but made them capital.

Reply:
I meant the change from `fo` to `file`.  I'm generally not happy with `black` complaining about short names like that.

Reply:
If it is bothering let's disable it at the TVM level. Otherwise we're gonna have bunch of `pylint_disable` tags which kind of defeats the purpose of pylint.